apps/api/src/app/workflows-v2/dto/workflow-commons-fields.ts

```diff
@@ -0,0 +1,76 @@
+import { IsArray, IsBoolean, IsDefined, IsObject, IsOptional, IsString } from 'class-validator';
+
+import { JsonSchema } from '@novu/framework';
+import { StepTypeEnum, WorkflowPreferences } from '@novu/shared';
+import { WorkflowResponseDto } from './workflow-response-dto';
+
+export class ControlsSchema {
+  schema: JsonSchema;
+}
+
+export type StepResponseDto = StepDto & {
+  stepUuid: string;
+};
+
+export type StepUpdateDto = StepDto & {
+  stepUuid: string;
+};
+
+export type StepCreateDto = StepDto;
+
+export type ListWorkflowResponse = {
+  workflows: WorkflowListResponseDto[];
+  totalResults: number;
```

Comment:
```suggestion
  totalCount: number;
```

Until now we used totalCount in the response, do you happen to be familiar with any best practices we could apply for the new API version?